sktime/transformations/series/dobin.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""Preprocessing algorithm DOBIN (Distance based Outlier BasIs using Neighbors)."""
+
+import warnings
+
+import numpy as np
+import pandas as pd
+from pandas.api.types import is_numeric_dtype
+from scipy.linalg import null_space
+from sklearn.decomposition import PCA
+from sklearn.neighbors import NearestNeighbors
+
+from sktime.transformations.base import BaseTransformer
+
+__author__ = ["KatieBuc"]
+__all__ = ["DOBIN"]
+
+
+class DOBIN(BaseTransformer):
+    """Distance based Outlier BasIs using Neighbors (DOBIN).
+
+    DOBIN is a pre-processing algorithm that constructs a set of basis
+    basistors tailored for outlier detection as described by _[1]. DOBIN
+    has a simple mathematical foundation and can be used as a dimension
+    reduction tool for outlier detection tasks.
+
+    Method assumes normalized data, for example:
+    from sklearn.preprocessing import MinMaxScaler, RobustScaler, StandardScaler
+
+    This prevents variables with large variances having disproportional
+    influence on Euclidean distances. The original implelemtation -[1] uses
+    MinMaxScaler normalization.
+
+    Parameters
+    ----------
+    frac : int, float (default=0.95)
+        The cut-off quantile for Y space
+    k : int (default=None)
+        Number of nearest neighbours considered, with a default value None calculated
+        as 5% of the number of observations with a cap of 20.
```

Comment:
Any chance you could reformulate what the default value is, it's a bit confusing.